deployment/I_exrm_releases.md

```diff
@@ -74,12 +74,13 @@ http: [port: 8888],
 url: [host: "example.com"],
 root: ".",
 cache_static_manifest: "priv/static/manifest.json",
-server: true
+server: true,
+version: Mix.Project.config[:version]
 ```
 
 When we run `mix phoenix.server` to start our application, the `server` parameter is automatically set to true. When we're creating a release, however, we need to configure this manually. If we get through this release guide, and we aren't seeing any pages coming from our server, this is a likely culprit.
 
-When generating a release and performing a hot upgrade, the static assets being served will not be the newest version without setting the root to `.`.  If you do not set root to `.` you will be forced to perform a restart so that the correct static assets are served, effectively changing the hot upgrade to a rolling restart.
+When generating a release and performing a hot upgrade, the static assets being served will not be the newest version without setting the root to `.` and setting the endpoint version.  If you do not set root or version you will be forced to perform a restart so that the correct static assets are served, effectively changing the hot upgrade to a rolling restart.
```

Comment:
How about:

 >  If `root` or `version` are not set then we will be forced...

Instead of 

 >  If you do not set root or version you will be forced...